README.md

```diff
@@ -17,32 +17,34 @@ The following steps should only be used to install if you are planning on contri
 1. Install [Node.js 16.x](https://nodejs.org/en/download/)
 1. Install [Rust](https://www.rust-lang.org/learn/get-started).
 1. Install [Yarn](https://classic.yarnpkg.com/en/docs/install).
+1. Run `yarn install` from the root directory to install packages (see sections below for OS specific details).
 1. Windows:
    1. Install [Build Tools for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers), including the C++ Build Tools and the Windows 10 SDK, for Rust.
    1. Rust builds also require installing [MinGW-w64 via win-builds](http://win-builds.org/doku.php/download_and_installation_from_windows).
       1. Choose `Native Windows`, `x86_64`, choose an empty directory, and click OK.
       1. On the next screen, click `Process`.
       1. Once it finishes, add the `bin` folder containing `cc` to your PATH environment variable.
-1. Run `yarn install` from the root directory to install packages.
-
+1. MacOS:
    - If `yarn install` fails with an error that includes "Failed to build cmake", you may need to first install cmake. For example, on macOS:
 
      1. Run `brew install cmake`, you'll need cmake version 3.15 or higher.
 
    - If `yarn install` fails with an error that includes "Could NOT find OpenSSL", you may need to first install OpenSSL and add an environment variable. For example, on macOS:
 
      1. Run `brew install openssl`
-     1. Run `` export OPENSSL_ROOT_DIR=`brew --prefix openssl`  ``
```

Comment:
Not sure which is preferable, I used the new line and it fixed permanently (no zshrc edits)